tests/docker/Dockerfile

```diff
@@ -113,7 +113,8 @@ RUN go install github.com/twmb/kcl@latest && \
 
 # Should fork to redpanda repo
 RUN git -C /opt clone https://github.com/jcsp/si-verifier.git && \
-    cd /opt/si-verifier && go mod tidy && go build
+    cd /opt/si-verifier && git reset --hard bcf4996d063e96497f8ff8be35d82fa1cec53180 && \
```

Comment:
Maybe move si-verifier to redpanda-data org? I forgot about it

Reply:
Yeah, I want to do that later after cleaning it up a bit & making sure the tool has the right interfaces for using it more tests.